packages/token-bridge-sdk/src/util/withdrawals.ts

```diff
@@ -0,0 +1,229 @@
+import { FetchTokenWithdrawalsFromSubgraphResult } from '../withdrawals/fetchTokenWithdrawalsFromSubgraph'
+import { Provider } from '@ethersproject/providers'
+import {
+  AssetType,
+  L2ToL1EventResult,
+  L2ToL1EventResultPlus,
+  NodeBlockDeadlineStatus,
+  OutgoingMessageState,
+  WithdrawalInitiated
+} from '../hooks/arbTokenBridge.types'
+import { getL1TokenData, isClassicL2ToL1TransactionEvent } from '../util'
+import { L2ToL1MessageReader, L2TransactionReceipt } from '@arbitrum/sdk'
+import { fetchL2BlockNumberFromSubgraph } from '../util/subgraph'
+
+export const outgoungStateToString = {
```

Comment:
It's already defined in `src/state/app/utils.ts`

Reply:
Done. Removed 1.